notebooks/04_inference.ipynb

```diff
@@ -196,22 +205,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 11,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Rescale the output resolution from 1m to 30m\n",
     "profile[\"transform\"][0] = 30\n",
     "profile[\"transform\"][4] = 30\n",
+    "profile[\"width\"] = profile[\"width\"] * (150 / 30)\n",
+    "profile[\"height\"] = profile[\"height\"] * (150 / 30)\n",
     "\n",
     "profile.update(\n",
-    "    dtype=np.float32,\n",
+    "    dtype=np.uint8,\n",
     "    count=1,\n",
     "    compress=\"DEFLATE\",\n",
     "    nodata=255,\n",
     ")\n",
     "\n",
-    "with rasterio.open(\"../data/pred/malheur_cnnv1_cancov_.tif\", \"w\", **profile) as dst:\n",
+    "with rasterio.open(f\"../data/pred/malheur_2016_{MODEL_ID}.tif\", \"w\", **profile) as dst:\n",
```

Comment:
Probably `MODEL_ID` here is sufficient.  In your examples, `MODEL_ID` already starts with `MAL2016`

Reply:
I agree that the current solution isn't great, but I'm a little hesitant to just go with `MODEL_ID` since the `MAL2016` model could potentially be used to generate a prediction for a different area or year. It seems to me like it should ultimately be identified by both the `MODEL_ID` and an ID associated with the NAIP data used for mapping, but I'm not quite sure how best to handle the latter. Let me know if you have any thoughts!

Reply:
Ahh, this was me not understanding that there was a difference between the first two elements (prediction area + year) and the model used for prediction.  Because this is in a notebook and gives the *example* flow, I'm not sure it's worth formalizing here.  As you say, coming up with an ID to uniquely identify the NAIP data would be useful, but not probably terribly important at this point. 

I confess to grabbing the NAIP 2016 window that covers the lidar acquisition just to see how it behaved where it was trained and didn't put any effort into differentiating the output prediction name.  Just appended a '2' on the end!

Reply:
Understandable - that distinction is a little ambiguous at the moment!

> I confess to grabbing the NAIP 2016 window that covers the lidar acquisition just to see how it behaved where it was trained

Interesting, how did it do? I've only compared the 2016 prediction to the 2010 and 2020 LiDAR datasets, which obviously aren't the best comparisons due to the temporal mismatch.

Reply:
> Interesting, how did it do? I've only compared the 2016 prediction to the 2010 and 2020 LiDAR datasets, which obviously aren't the best comparisons due to the temporal mismatch.

Patterns look really good to me in terms of capturing fine scale landscape features.  There looks to be an attenuation effect (i.e. regression to the mean) where it's not capturing the highest highs / lowest lows, but that's definitely not uncommon for a lot of modeling.

| Lidar canopy cover | Predicted NAIP canopy cover |
| ----------- | ----------- |
| <img src="https://github.com/lemma-osu/naip-cnn/assets/1706997/1276313c-9c86-4309-bb69-6f88dfabf875" width="400" height="400"> | <img src="https://github.com/lemma-osu/naip-cnn/assets/1706997/be5c34d4-7659-449a-998d-3d4381a499df" width="400" height="400">

Reply:
Glad to hear! I'm sure there's still room for improvement in the model and training parameters, so hopefully we can match the distributions a little better before we're done.